python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1088,7 +1089,8 @@ async def move_m2_hexapod(
     ) -> None:
         """Move m2 hexapod.
 
-        When m2 hexapod compensation mode is on move will act as offset.
+        When m2 hexapod compensation mode is on, move is with respect to
+        the LUT value.
```

Comment:
My suggestion on the `move_camera_hexapod` also applies here, in case it is valid.